### PR TITLE
[WIP] Update of README

### PR DESCRIPTION
**Pull request by aboucaud**
I started cleaning up the commands.

I also removed the Makefile and placed it in .gitignore.
This way @kegl can keep the Makefile in his local folder 😄 but I think it is not needed anymore.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,30 +57,35 @@ If you **have a predictive problem**, you can **submit it as a data challenge**
...
 
-Starting kits in [ramp-kits][rkits] are working workflows and workflow instantiations. They work out of the box. You can run them using the `ramp_test_submission` script that simply executes [`testing_py`](rampwf/utils/testing.py) in the starting kit. For example, clone the titanic starting kit and test it by
+Starting kits in [ramp-kits][rkits] are working workflows and workflow instantiations. They work out of the box. You can run them using the `ramp_test_submission` script that simply executes [`testing_py`](rampwf/utils/testing.py) in the starting kit. For example, start by cloning the starting kit (here _titanic_) in the repository of your choosing
+
+```bash
+$ cd /path/to/your/ramp-kits/repo
+$ git clone https://github.com/ramp-kits/titanic.git
+```
+
+and test the default submission by moving into the repo and running the `ramp_test_submission` script
```
Comment (kegl) [inline README.md]: Shouldn't we pip install requirements before launching `ramp_test_submission`?

File: README.md [hunk 2]
```diff
@@ -60,35 +57,53 @@ If you **have a predictive problem**, you can **submit it as a data challenge**
...
 ### Get familiar with starting kits
 
-Starting kits in [ramp-kits][rkits] are working workflows and workflow instantiations. They work out of the box. You can run them using the `ramp_test_submission` script that simply executes [`testing_py`](rampwf/utils/testing.py) in the starting kit. For example, clone the titanic starting kit and test it by
+Starting kits in [ramp-kits][rkits] are working workflows and workflow instantiations. 
+They work out of the box. 
+
+Start by cloning the starting kit `<ramp_kit_name>` in the repository of your choosing
+
+```bash
+$ cd /path/to/your/ramp-kits/repo
```
Comment (jorisvandenbossche) [inline README.md]: I would replace 'repo' with eg 'directory' as the ramp-kits is not a repo, you are cloning a repo with the next step

**Comment by jorisvandenbossche** [comment]
Looks good

**Comment by kegl** [comment]
To me too.

**Pull request merged**